examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/stm32_xx_it.c

```diff
@@ -69,6 +70,38 @@ void TIMp_IRQHandler(void);
 /* Private functions ---------------------------------------------------------*/
 
 /******************************************************************************/
+/*                Demo GPIO Interrupt implementation                         */
+/******************************************************************************/

```

Comment:
@atigyi I just pushed the fix for the button interrupt. This is the biggest change, the rest of the modifications were for style or clean-up. If these 2 functions look good to you, I'll merge the branch :)